README.md

```diff
@@ -1,5 +1,18 @@
 ## Preface
 <!-- Optional, the preface is not about the subject -->
+An analysis of WorldSkills Russia 
+competition events 
+and demonstration exams
```

Comment:
I've tried to distinguish demonstration exams and looks like we will look into competition events with only , since we can't identify a significant amount of exam results (only 187 of 253k entries).

Reply:
Got it, changed it to just competition events.